framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -73,18 +86,26 @@ export const ComboPicker = ({
     }
   }, [ enableSelectInput ])
 
+  useEffect(() => {
+    if (!is(Number, inputValue) && defaultToZeroIfEmpty) {
+      handleInputChange(clamp(min, max, 0))
+    }
+  }, [ inputValue ])
```

Comment:
Hmmm handleInputChange calls setInputValueState, which means that if the value is uncontrolled this could turn into a loop, maybe the useEffect dependency should only be valueProp.input?